### DISCUSSION TITLE
Do I need a license for making a web tv channel??

### DISCUSSION THREAD

**Discussion by yumnycheesecakes** [savonet/liquidsoap#3105] [Q&A]
Hello. Do I need a license for making a web tv channel??

**Comment by toots** [comment]
Hi! Not from us! We're a free and open source project. 

The liquidsoap code is free for use, modify and distribute following the terms of the GNU GPL license: https://github.com/savonet/liquidsoap/blob/main/COPYING

The liquidsoap application also relies on third-party code, all of it also being free to use at least, the rest of it depending on their respective licensing terms.

You might need to consider licenses for the content that you want to stream or the encoding technology used to provide your content online.
